server/app/services/applicant/question/EmailQuestion.java

```diff
@@ -33,6 +35,16 @@ protected ImmutableMap<Path, ImmutableSet<ValidationErrorMessage>> getValidation
     return ImmutableMap.of();
   }
 
+  @Override
+  public ImmutableMap<Path, String> getJsonEntries() {
+    return applicantQuestion.getContextualizedScalars().keySet().stream()
+        .filter(path -> !Scalar.getMetadataScalarKeys().contains(path.keyName()))
+        .collect(
+            toImmutableMap(
+                path -> path,
+                path -> applicantQuestion.getApplicantData().readAsString(path).orElse("")));
+  }
+
   @Override
```

Comment:
Above code is taken from default case:

https://github.com/civiform/civiform/blob/358ad3dbb6e67aacb91afb3ede60e32ac281ba26/server/app/services/export/JsonExporter.java#L121-L219

Which in turn uses the default case here:

https://github.com/civiform/civiform/blob/358ad3dbb6e67aacb91afb3ede60e32ac281ba26/server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java#L450-L508